packages/client/src/execution/vmexecution.ts

```diff
@@ -39,6 +43,8 @@ export class VMExecution extends Execution {
   private _lock = new Lock()
 
   public vm: VM
+  public merkleVm: VM | null = null
+  public verkleVm: VM | null = null
```

Comment:
Do we need these `null` assignments here both for the verkle and the merkle VM objects and have this ambiguity with `undefined` and `null` here again or can we just stick with `undefined`?

Reply:
Am also not sure why we're going with `null` instead of undefined. @g11tech ?

Reply:
Can you maybe directly trie to update (aka: remove `null`) if Gajinder does not intervene in the meantime 🙂 and you also see no counter-arguments as well?